hbase-client/src/main/java/org/apache/hadoop/hbase/client/Append.java

```diff
@@ -162,10 +162,9 @@ public Append addColumn(byte[] family, byte[] qualifier, byte[] value) {
 
   /**
    * Add column and value to this Append operation.
-   * @param cell
    * @return This instance
    */
-  @SuppressWarnings("unchecked")
+  @Override
```

Comment:
Seems to be unrelated to the actual change.

Reply:
Just some minor fix on checkstyle.